package.json

```diff
@@ -1,89 +1,93 @@
-    "babel-eslint": "^10.0.3",
-    "chai": "^4.1.2",
-    "commitizen": "^4.2.1",
-    "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^5.16.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-vue": "^5.2.3",
-    "lint-staged": "^8.1.5",
-    "mem": "^5.1.1",
-    "sass": "^1.25.0",
-    "sass-loader": "^7.1.0",
-    "vue-template-compiler": "^2.6.11"
-  },
-  "eslintConfig": {
-    "root": true,
-    "extends": [
-      "plugin:vue/essential",
-      "plugin:prettier/recommended",
-      "eslint:recommended"
-    ]
-  },
-  "gitHooks": {
-    "pre-commit": "lint-staged"
-  },
-  "lint-staged": {
-    "*.{js,vue}": [
-      "vue-cli-service lint",
-      "git add"
-    ]
-  }
+	"name": "responsive-navigation",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"serve": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "vue-cli-service lint",
+		"start": "node server.js",
+		"test": "vue-cli-service test:unit",
+		"commit": "cz",
+		"format": "prettier --write \"src/**/*.{js,vue,json,css,md}\""
+	},
+	"dependencies": {
+		"@ckeditor/ckeditor5-build-classic": "^18.0.0",
+		"@ckeditor/ckeditor5-vue": "^1.0.1",
+		"@zoomus/websdk": "^1.7.5",
+		"axios": "^0.19.2",
+		"body-parser": "^1.19.0",
+		"bootstrap": "^4.4.1",
+		"bootstrap-vue": "^2.4.0",
+		"capitalize": "^2.0.1",
+		"compression": "^1.7.4",
+		"cookie-parser": "^1.4.4",
+		"core-js": "^2.6.11",
+		"dotenv": "^8.2.0",
+		"express": "^4.17.1",
+		"express-session": "^1.17.0",
+		"heroku-ssl-redirect": "0.0.4",
+		"moment-timezone": "^0.5.27",
+		"react": "^16.13.1",
+		"react-dom": "^16.13.1",
+		"react-redux": "^7.2.0",
+		"redux": "^4.0.5",
+		"redux-thunk": "^2.3.0",
+		"register-service-worker": "^1.7.1",
+		"uuid": "^8.0.0",
+		"vue": "^2.6.11",
+		"vue-authenticate": "^1.4.1",
+		"vue-cookies": "^1.6.1",
+		"vue-country-region-select": "^2.0.11",
+		"vue-infinite-scroll": "^2.0.2",
+		"vue-markdown": "^2.2.4",
+		"vue-moment": "^4.1.0",
+		"vue-router": "^3.1.5",
+		"vue-star-rating": "^1.6.1",
+		"vue-tweet-embed": "^2.4.0",
+		"vue-youtube": "^1.4.0",
+		"vuex": "^3.1.2"
+	},
+	"devDependencies": {
+		"@vue/cli-plugin-babel": "^3.12.1",
+		"@vue/cli-plugin-eslint": "^3.12.1",
+		"@vue/cli-plugin-pwa": "^4.4.6",
+		"@vue/cli-plugin-unit-mocha": "^3.12.1",
+		"@vue/cli-service": "^3.12.1",
+		"@vue/eslint-config-standard": "^4.0.0",
+		"@vue/test-utils": "1.0.0-beta.29",
+		"babel-eslint": "^10.0.3",
+		"chai": "^4.1.2",
+		"commitizen": "^4.2.1",
+		"cz-conventional-changelog": "^3.3.0",
+		"eslint": "^5.16.0",
+		"eslint-config-prettier": "^6.11.0",
+		"eslint-plugin-prettier": "^3.1.3",
+		"eslint-plugin-vue": "^5.2.3",
+		"lint-staged": "^8.1.5",
+		"mem": "^5.1.1",
+		"sass": "^1.25.0",
+		"sass-loader": "^7.1.0",
+		"vue-template-compiler": "^2.6.11"
+	},
+	"eslintConfig": {
+		"root": true,
+		"extends": [
+			"plugin:vue/essential",
+			"plugin:prettier/recommended",
```

Comment:
since we are using eslint prettier plugin, do we need to introduce a separate .prettierrc config?

Reply:
Thank you for bringing this up.

This was not easy find but seems like I spotted something which can be helpful. In [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier#recommended-configuration) they mention to keep `.prettierrc` file.

Quote from their repository - 
> Note: While it is possible to pass options to Prettier via your ESLint configuration file, it is not recommended because editor extensions such as prettier-atom and prettier-vscode will read .prettierrc, but won't read settings from ESLint, which can lead to an inconsistent experience.

Reply:
I'm not completely satisfied with this PR, things are not smooth.
Let's hold this one until I make it better.

Reply:
I believe we are good now and things are working as expected.
Seems like it was an issue with WebStorm. Performed multiple tests and didn't face issue.

Few things helped me - 
- https://github.com/okonet/lint-staged#configuration
- https://github.com/okonet/lint-staged/issues/599

We are good to go.